src/Umbraco.Core/Security/BackOfficeSignInManager.cs

```diff
@@ -250,5 +250,42 @@ public static BackOfficeSignInManager Create(IdentityFactoryOptions<BackOfficeSi
             }
             return null;
         }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// Two factor verification step
+        /// </summary>
+        /// <param name="provider"></param>
+        /// <param name="code"></param>
+        /// <param name="isPersistent"></param>
+        /// <param name="rememberBrowser"></param>
+        /// <returns></returns>
+        public override async Task<SignInStatus> TwoFactorSignInAsync(string provider, string code, bool isPersistent, bool rememberBrowser)
+        {
+            var userId = await GetVerifiedUserIdAsync();
+            if (userId == null)
```

Comment:
in this case this can never be null since we're only ever dealing with INT which is a struct. I'll leave the check in anyways as it wont cause any harm.  I've verified this code is the same as the underlying code so all good.

Reply:
Actually this needs to check for -1